pkg/apis/core/types.go

```diff
@@ -3000,14 +3000,12 @@ type PodSpec struct {
 	// - spec.containers[*].securityContext.runAsGroup
 	// +optional
 	OS *PodOS
-	// Each scheduling gate represents a particular scenario the scheduling is blocked upon.
-	// Scheduling is triggered only when SchedulingGates is empty.
+	// Each scheduling gate represents a particular scenario that must be satisfied before scheduling.
```

Comment:
Nit: can we be consistent with the description we have for finalizers?

"// schedulingGates is an opaque list of values that if specified will block scheduling the pod."